dev-requirements.txt

```diff
@@ -1,7 +1 @@
-.
-pytest>=7.0
-pytest-xdist[psutil]
-wheel
-betamax>=0.5.1
-betamax_matchers>=0.3.0
-tox>=3.1.3
+.[dev]
```

Comment:
Isn't there a command to install dependencies with dev so we don't need this file?

Reply:
`pip install -e .[dev]` is the equivalent. Honestly, this file could be deleted.